openshift-hack/build-rpm-rhel8.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -x
+
+source_path=_output/SOURCES
+
+mkdir -p ${source_path}
+
+# Getting version from args
+version=${1:-4.10.0}
```

Comment:
Start with a more up to date default? 4.14 since that's the intended backport target?